src/SmartComponents/RunTaskModal/ConversionTaskInputParameters.js

```diff
@@ -175,6 +187,37 @@ const ConversionTaskInputParameters = ({
         skipPackageCheckValue,
         setSkipPackageCheckValue
       )}
+
+      <FormGroup role="group" fieldId="optional-repositories">
```

Comment:
I feel this Form can be extracted into a separate file with its own state and logic.